packages/data-explorer-ui/src/components/common/Button/components/CallToActionButton/callToActionButton.stories.tsx

```diff
@@ -1,6 +1,7 @@
-import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Meta, StoryFn } from "@storybook/react";
```

Comment:
```suggestion
import { Meta, StoryObj } from "@storybook/react";
```

Reply:
Would it be possible to revert the CallToActionButton change requests altogether?